2023-10-26 10:23:55.178 ERROR org.quartz.core.JobRunShell                       :211  schedulerFactoryBean_Worker-1 [32m1698287035111b5e[0;39m Job DEFAULT.com.example.train.batch.job.DailyTrainJob threw an unhandled Exception: 
feign.FeignException$ServiceUnavailable: [503] during [GET] to [http://business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [Load balancer does not contain an instance for the service business]
	at feign.FeignException.serverErrorStatus(FeignException.java:265)
	at feign.FeignException.errorStatus(FeignException.java:206)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy4/jdk.proxy4.$Proxy99.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
2023-10-26 10:23:55.183 ERROR org.quartz.core.ErrorLogger                       :2407 schedulerFactoryBean_Worker-1 [32m1698287035111b5e[0;39m Job (DEFAULT.com.example.train.batch.job.DailyTrainJob threw an exception.
org.quartz.SchedulerException: Job threw an unhandled exception.
	at org.quartz.core.JobRunShell.run(JobRunShell.java:213)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
Caused by: feign.FeignException$ServiceUnavailable: [503] during [GET] to [http://business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [Load balancer does not contain an instance for the service business]
	at feign.FeignException.serverErrorStatus(FeignException.java:265)
	at feign.FeignException.errorStatus(FeignException.java:206)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy4/jdk.proxy4.$Proxy99.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	... 1 common frames omitted
2023-10-26 10:24:57.245 ERROR org.quartz.core.JobRunShell                       :211  schedulerFactoryBean_Worker-2 [32m1698287097243cwu[0;39m Job DEFAULT.com.example.train.batch.job.DailyTrainJob threw an unhandled Exception: 
feign.FeignException$ServiceUnavailable: [503] during [GET] to [http://business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [Load balancer does not contain an instance for the service business]
	at feign.FeignException.serverErrorStatus(FeignException.java:265)
	at feign.FeignException.errorStatus(FeignException.java:206)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy4/jdk.proxy4.$Proxy99.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
2023-10-26 10:24:57.246 ERROR org.quartz.core.ErrorLogger                       :2407 schedulerFactoryBean_Worker-2 [32m1698287097243cwu[0;39m Job (DEFAULT.com.example.train.batch.job.DailyTrainJob threw an exception.
org.quartz.SchedulerException: Job threw an unhandled exception.
	at org.quartz.core.JobRunShell.run(JobRunShell.java:213)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
Caused by: feign.FeignException$ServiceUnavailable: [503] during [GET] to [http://business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [Load balancer does not contain an instance for the service business]
	at feign.FeignException.serverErrorStatus(FeignException.java:265)
	at feign.FeignException.errorStatus(FeignException.java:206)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy4/jdk.proxy4.$Proxy99.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	... 1 common frames omitted
2023-10-26 10:27:25.014 ERROR org.quartz.core.JobRunShell                       :211  schedulerFactoryBean_Worker-3 [32m1698287245013phh[0;39m Job DEFAULT.com.example.train.batch.job.DailyTrainJob threw an unhandled Exception: 
feign.FeignException$ServiceUnavailable: [503] during [GET] to [http://business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [Load balancer does not contain an instance for the service business]
	at feign.FeignException.serverErrorStatus(FeignException.java:265)
	at feign.FeignException.errorStatus(FeignException.java:206)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy4/jdk.proxy4.$Proxy99.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
2023-10-26 10:27:25.015 ERROR org.quartz.core.ErrorLogger                       :2407 schedulerFactoryBean_Worker-3 [32m1698287245013phh[0;39m Job (DEFAULT.com.example.train.batch.job.DailyTrainJob threw an exception.
org.quartz.SchedulerException: Job threw an unhandled exception.
	at org.quartz.core.JobRunShell.run(JobRunShell.java:213)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
Caused by: feign.FeignException$ServiceUnavailable: [503] during [GET] to [http://business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [Load balancer does not contain an instance for the service business]
	at feign.FeignException.serverErrorStatus(FeignException.java:265)
	at feign.FeignException.errorStatus(FeignException.java:206)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy4/jdk.proxy4.$Proxy99.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	... 1 common frames omitted
2023-10-26 10:27:30.014 ERROR org.quartz.core.JobRunShell                       :211  schedulerFactoryBean_Worker-4 [32m1698287250012469[0;39m Job DEFAULT.com.example.train.batch.job.DailyTrainJob threw an unhandled Exception: 
feign.FeignException$ServiceUnavailable: [503] during [GET] to [http://business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [Load balancer does not contain an instance for the service business]
	at feign.FeignException.serverErrorStatus(FeignException.java:265)
	at feign.FeignException.errorStatus(FeignException.java:206)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy4/jdk.proxy4.$Proxy99.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
2023-10-26 10:27:30.014 ERROR org.quartz.core.ErrorLogger                       :2407 schedulerFactoryBean_Worker-4 [32m1698287250012469[0;39m Job (DEFAULT.com.example.train.batch.job.DailyTrainJob threw an exception.
org.quartz.SchedulerException: Job threw an unhandled exception.
	at org.quartz.core.JobRunShell.run(JobRunShell.java:213)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
Caused by: feign.FeignException$ServiceUnavailable: [503] during [GET] to [http://business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [Load balancer does not contain an instance for the service business]
	at feign.FeignException.serverErrorStatus(FeignException.java:265)
	at feign.FeignException.errorStatus(FeignException.java:206)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy4/jdk.proxy4.$Proxy99.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	... 1 common frames omitted
2023-10-26 10:27:35.014 ERROR org.quartz.core.JobRunShell                       :211  schedulerFactoryBean_Worker-5 [32m1698287255013x0u[0;39m Job DEFAULT.com.example.train.batch.job.DailyTrainJob threw an unhandled Exception: 
feign.FeignException$ServiceUnavailable: [503] during [GET] to [http://business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [Load balancer does not contain an instance for the service business]
	at feign.FeignException.serverErrorStatus(FeignException.java:265)
	at feign.FeignException.errorStatus(FeignException.java:206)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy4/jdk.proxy4.$Proxy99.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
2023-10-26 10:27:35.014 ERROR org.quartz.core.ErrorLogger                       :2407 schedulerFactoryBean_Worker-5 [32m1698287255013x0u[0;39m Job (DEFAULT.com.example.train.batch.job.DailyTrainJob threw an exception.
org.quartz.SchedulerException: Job threw an unhandled exception.
	at org.quartz.core.JobRunShell.run(JobRunShell.java:213)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
Caused by: feign.FeignException$ServiceUnavailable: [503] during [GET] to [http://business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [Load balancer does not contain an instance for the service business]
	at feign.FeignException.serverErrorStatus(FeignException.java:265)
	at feign.FeignException.errorStatus(FeignException.java:206)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy4/jdk.proxy4.$Proxy99.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	... 1 common frames omitted
2023-10-26 10:27:40.013 ERROR org.quartz.core.JobRunShell                       :211  schedulerFactoryBean_Worker-6 [32m1698287260012z08[0;39m Job DEFAULT.com.example.train.batch.job.DailyTrainJob threw an unhandled Exception: 
feign.FeignException$ServiceUnavailable: [503] during [GET] to [http://business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [Load balancer does not contain an instance for the service business]
	at feign.FeignException.serverErrorStatus(FeignException.java:265)
	at feign.FeignException.errorStatus(FeignException.java:206)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy4/jdk.proxy4.$Proxy99.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
2023-10-26 10:27:40.014 ERROR org.quartz.core.ErrorLogger                       :2407 schedulerFactoryBean_Worker-6 [32m1698287260012z08[0;39m Job (DEFAULT.com.example.train.batch.job.DailyTrainJob threw an exception.
org.quartz.SchedulerException: Job threw an unhandled exception.
	at org.quartz.core.JobRunShell.run(JobRunShell.java:213)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
Caused by: feign.FeignException$ServiceUnavailable: [503] during [GET] to [http://business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [Load balancer does not contain an instance for the service business]
	at feign.FeignException.serverErrorStatus(FeignException.java:265)
	at feign.FeignException.errorStatus(FeignException.java:206)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy4/jdk.proxy4.$Proxy99.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	... 1 common frames omitted
2023-10-26 10:52:04.710 ERROR org.quartz.core.JobRunShell                       :211  schedulerFactoryBean_Worker-1 [32m1698288724670mom[0;39m Job DEFAULT.com.example.train.batch.job.DailyTrainJob threw an unhandled Exception: 
feign.FeignException$NotFound: [404] during [GET] to [http://127.0.0.1:8002/business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [{"timestamp":"2023-10-26T02:52:04.694+00:00","status":404,"error":"Not Found","message":"No message available","path":"/business/business/admin/daily-train/gen-daily/2023-11-10"}]
	at feign.FeignException.clientErrorStatus(FeignException.java:228)
	at feign.FeignException.errorStatus(FeignException.java:203)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy7/jdk.proxy7.$Proxy106.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
2023-10-26 10:52:04.712 ERROR org.quartz.core.ErrorLogger                       :2407 schedulerFactoryBean_Worker-1 [32m1698288724670mom[0;39m Job (DEFAULT.com.example.train.batch.job.DailyTrainJob threw an exception.
org.quartz.SchedulerException: Job threw an unhandled exception.
	at org.quartz.core.JobRunShell.run(JobRunShell.java:213)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
Caused by: feign.FeignException$NotFound: [404] during [GET] to [http://127.0.0.1:8002/business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [{"timestamp":"2023-10-26T02:52:04.694+00:00","status":404,"error":"Not Found","message":"No message available","path":"/business/business/admin/daily-train/gen-daily/2023-11-10"}]
	at feign.FeignException.clientErrorStatus(FeignException.java:228)
	at feign.FeignException.errorStatus(FeignException.java:203)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy7/jdk.proxy7.$Proxy106.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	... 1 common frames omitted
2023-10-26 10:52:45.730 ERROR org.quartz.core.JobRunShell                       :211  schedulerFactoryBean_Worker-2 [32m1698288765720fla[0;39m Job DEFAULT.com.example.train.batch.job.DailyTrainJob threw an unhandled Exception: 
feign.FeignException$NotFound: [404] during [GET] to [http://127.0.0.1:8002/business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [{"timestamp":"2023-10-26T02:52:45.728+00:00","status":404,"error":"Not Found","message":"No message available","path":"/business/business/admin/daily-train/gen-daily/2023-11-10"}]
	at feign.FeignException.clientErrorStatus(FeignException.java:228)
	at feign.FeignException.errorStatus(FeignException.java:203)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy7/jdk.proxy7.$Proxy106.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
2023-10-26 10:52:45.730 ERROR org.quartz.core.ErrorLogger                       :2407 schedulerFactoryBean_Worker-2 [32m1698288765720fla[0;39m Job (DEFAULT.com.example.train.batch.job.DailyTrainJob threw an exception.
org.quartz.SchedulerException: Job threw an unhandled exception.
	at org.quartz.core.JobRunShell.run(JobRunShell.java:213)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
Caused by: feign.FeignException$NotFound: [404] during [GET] to [http://127.0.0.1:8002/business/business/admin/daily-train/gen-daily/2023-11-10] [BusinessFeign#genDaily(Date)]: [{"timestamp":"2023-10-26T02:52:45.728+00:00","status":404,"error":"Not Found","message":"No message available","path":"/business/business/admin/daily-train/gen-daily/2023-11-10"}]
	at feign.FeignException.clientErrorStatus(FeignException.java:228)
	at feign.FeignException.errorStatus(FeignException.java:203)
	at feign.FeignException.errorStatus(FeignException.java:194)
	at feign.codec.ErrorDecoder$Default.decode(ErrorDecoder.java:104)
	at feign.ResponseHandler.decodeError(ResponseHandler.java:136)
	at feign.ResponseHandler.handleResponse(ResponseHandler.java:70)
	at feign.SynchronousMethodHandler.executeAndDecode(SynchronousMethodHandler.java:114)
	at feign.SynchronousMethodHandler.invoke(SynchronousMethodHandler.java:70)
	at feign.ReflectiveFeign$FeignInvocationHandler.invoke(ReflectiveFeign.java:96)
	at jdk.proxy7/jdk.proxy7.$Proxy106.genDaily(Unknown Source)
	at com.example.train.batch.job.DailyTrainJob.execute(DailyTrainJob.java:35)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	... 1 common frames omitted
2023-10-26 10:52:45.750 ERROR o.s.scheduling.quartz.LocalDataSourceJobStore     :3729 schedulerFactoryBean_Worker-2 [32m1698288765720fla[0;39m Couldn't rollback jdbc connection. Connection is closed
java.sql.SQLException: Connection is closed
	at com.zaxxer.hikari.pool.ProxyConnection$ClosedConnection.lambda$getClosedConnection$0(ProxyConnection.java:502)
	at jdk.proxy3/jdk.proxy3.$Proxy79.rollback(Unknown Source)
	at com.zaxxer.hikari.pool.ProxyConnection.rollback(ProxyConnection.java:385)
	at com.zaxxer.hikari.pool.HikariProxyConnection.rollback(HikariProxyConnection.java)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.quartz.impl.jdbcjobstore.AttributeRestoringConnectionInvocationHandler.invoke(AttributeRestoringConnectionInvocationHandler.java:73)
	at jdk.proxy7/jdk.proxy7.$Proxy105.rollback(Unknown Source)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.rollbackConnection(JobStoreSupport.java:3727)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.executeInNonManagedTXLock(JobStoreSupport.java:3868)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.retryExecuteInNonManagedTXLock(JobStoreSupport.java:3819)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.triggeredJobComplete(JobStoreSupport.java:3116)
	at org.quartz.core.QuartzScheduler.notifyJobStoreJobComplete(QuartzScheduler.java:1786)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:269)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
2023-10-26 10:52:45.750 ERROR o.s.scheduling.quartz.LocalDataSourceJobStore     :3729 schedulerFactoryBean_Worker-2 [32m1698288765720fla[0;39m Couldn't rollback jdbc connection. Connection is closed
java.sql.SQLException: Connection is closed
	at com.zaxxer.hikari.pool.ProxyConnection$ClosedConnection.lambda$getClosedConnection$0(ProxyConnection.java:502)
	at jdk.proxy3/jdk.proxy3.$Proxy79.rollback(Unknown Source)
	at com.zaxxer.hikari.pool.ProxyConnection.rollback(ProxyConnection.java:385)
	at com.zaxxer.hikari.pool.HikariProxyConnection.rollback(HikariProxyConnection.java)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.quartz.impl.jdbcjobstore.AttributeRestoringConnectionInvocationHandler.invoke(AttributeRestoringConnectionInvocationHandler.java:73)
	at jdk.proxy7/jdk.proxy7.$Proxy105.rollback(Unknown Source)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.rollbackConnection(JobStoreSupport.java:3727)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.executeInNonManagedTXLock(JobStoreSupport.java:3886)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.retryExecuteInNonManagedTXLock(JobStoreSupport.java:3819)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.triggeredJobComplete(JobStoreSupport.java:3116)
	at org.quartz.core.QuartzScheduler.notifyJobStoreJobComplete(QuartzScheduler.java:1786)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:269)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
2023-10-26 10:53:00.758 ERROR org.quartz.simpl.SimpleThreadPool                 :585  schedulerFactoryBean_Worker-2 [32m1698288765720fla[0;39m Error while executing the Runnable: 
java.lang.IllegalStateException: JobStore is shutdown - aborting retry
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.retryExecuteInNonManagedTXLock(JobStoreSupport.java:3833)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.triggeredJobComplete(JobStoreSupport.java:3116)
	at org.quartz.core.QuartzScheduler.notifyJobStoreJobComplete(QuartzScheduler.java:1786)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:269)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
